flopy/pakbase.py

```diff
@@ -675,6 +675,14 @@ def export(self, f, **kwargs):
 
         return export.utils.package_export(f, self, **kwargs)
 
+    def _generate_heading(self):
+        """Generate heading."""
+        parent = self.parent
+        self.heading = (
+            f"# {self.name[0]} package for "
+            f"{parent.version_types[parent.version]} generated by Flopy."
```

Comment:
We should add the Flopy version 

`... generated by Flopy version {__version__}"`

Reply:
Done